scss/civicrm/_mixins.scss

```diff
@@ -36,4 +36,69 @@
 
 @mixin translate ($x, $y) {
   @include transform(translate($x, $y));
+}
+
+/*
+* The styled-checkbox() CSS function designs individual checkbox
+*/
+
+@mixin styled-checkbox() {
+  height: 16px;
+  position: relative;
+
+  &::before {
+    content: '';
```

Comment:
please list the properties in alphabetical order (this is a general rule, if you haven't followed it so far, please start now)